internal/pkg/client/unstructured/helpers.go

```diff
@@ -0,0 +1,106 @@
+/*
```

Comment:
This is useful.  Can you add a doc.go explaining this library.

Reply:
Will done once i refactor the code using jsonpath